<!DOCTYPE html>
<html>
<head>
    <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Antoine Szatkownik</title>

    <link rel="stylesheet" href="css/main.css" media="screen" />
    <link rel="stylesheet" href="css/main-mobile.css" media="screen" />
    <link rel="stylesheet" href="css/home.css" media="screen" />
    <link rel="stylesheet" href="css/home-mobile.css" media="screen" />
    <link rel="stylesheet" href="css/main-print.css" media="print" />
    <link rel="stylesheet" href="css/home-print.css" media="print" />

    <link rel="shortcut icon" href="images/icon.png"/>

</head>
<body>



<header>
    <div class="colonne">
        <a href="home.html">Antoine Szatkownik</a> <br><br>
        Interested in patterns <br> Machine Learning – Computational Biology
    </div>
    <div class="colonne">
        <p class="screen">[<a href="https://github.com/antoineszatkownik" target="blank">Github</a>]</p>
        <p class="screen">[<a href="https://scholar.google.com/citations?user=UmRH2XYAAAAJ&hl=fr&oi=ao" target="blank">Google Scholar</a>]</p>
        <p class="screen">[<a href="https://www.linkedin.com/in/antoine-szatkownik-a61067220/" target="blank">Linkedin</a>]</p>
        <p class="screen">[<a href="gallery.html">Personal gallery</a>]</p>
        

        <a class="print">October 30th 1997</a>
        <a class="print">antoine.szatkownik@gmail.com</a>
        <a class="print">www.github.com/antoineszatkownik/</a>    
    </div>
</header>

<main>

    <div class="colonne">
        <div class="block">
            <h1>Experiences</h1>
            <div class="line">
                <p class="date">
                    <span>2022</span><br> 
                    (6 months)<br>
                    Berlin, GE
                </p>
                <p class="description">
                    <strong>Robert Koch Institute (RKI)</strong><br>
                    Master internship joint with LCQB<br>
                    <em class="small">Subject: Development of a computational method exploiting the interplay between gene duplication and alternative splicing to learn about the determinants of molecular recognition specificity</em><br>
                    <em class="small">Supervisors: Hugues Richard and <a href="https://www.lcqb.upmc.fr/laine/" target="_blank"> Elodie Laine</a> </em>
                </p>
            </div>
            <div class="line">
                <p class="date">
                    <span>2021</span> <br> 
                    (4 months)<br> 
                    Paris, FR
                </p>
                <p class="description">
                    <strong>Laboratory of Computational and Quantitative Biology (LCQB)</strong><br>
                    Master internship<br>
                    <em class="small">Title: Discovery of protein specificity signatures in evolutionary splicing graphs</em><br>
                    <em class="small">Supervisors: <a href="https://www.lcqb.upmc.fr/laine/" target="_blank"> Elodie Laine</a> and Hugues Richard</em>
                </p>
            </div>
        </div>
        

        <div class="block">
            <h1>Education</h1>
            <div class="line">
                <p class="date">
                    <span>2022 – 2025</span><br> 
                    Gif-sur-Yvette, FR
                </p>
                <p class="description">
                    <strong>Université Paris-Saclay</strong><br>
                    PhD in Computer Science, Laboratoire Interdisciplinaire des Sciences du Numérique (LISN)<br>
                    <em class="small">Title: Latent generative modeling and synthetic data evaluation in population genomics</em><br>
                    <em class="small">Supervisors: Flora Jay and <a href="https://www.lri.fr/~gcharpia/" target="_blank"> Guillaume Charpiat</a> </em><br>
                    <em class="small">Co-supervisors: Burak Yelmen and Cyril Furtlehner</em>
                </p>
            </div>
            <div class="line">
                <p class="date">
                    <span>2020 – 2022</span> <br> 
                    Paris, FR
                </p>
                <p class="description">
                    <strong>Sorbonne Université</strong><br>
                    Master, Specialization in Bioinformatics and Modeling (BIM)
                </p>
            </div>            
            <div class="line">
                <p class="date">
                    <span>2018 – 2020</span> <br> 
                    Orsay, FR
                </p>
                <p class="description">
                    <strong>Université Paris-Sud</strong><br>
                    Bachelor, Fundamentals and Applied Mathematics (MFA)
                </p>
            </div>    
            <div class="line">
                <p class="date">
                    <span>2016 – 2018</span> <br> 
                    Montréal, CN
                </p>
                <p class="description">
                    <strong>Université de Montréal</strong><br>
                    Bachelor, Bidisciplinary in Mathematics and Physics
                </p>
            </div>   
        </div>

        <div class="block">
            <h1>Teaching</h1>
            <div class="line">
                <p class="date">
                    <span>2023 – 2024</span><br> 
                    Orsay, FR
                </p>
                <p class="description">
                    <strong>MVA ENS + CentraleSupelec + DSBA ESSEC</strong><br>
                    <em class="small">Course: Deep Learning in Practice - M2 level</em><br>
                    <em class="small">Lecturer: Guillaume Charpiat</em><br>
                    <em class="small">Position: Teaching assistant (24h per academic year)</em>
                </p>
            </div>
            <div class="line">
                <p class="date">
                    <span>2022 – 2024</span> <br> 
                    Orsay, FR
                </p>
                <p class="description">
                    <strong>Université Paris-Saclay</strong><br>
                    <em class="small">Course: Introduction to machine learning - 3rnd year BCompSc level</em><br>
                    <em class="small">Lecturer: François Landes</em><br>
                    <em class="small">Position: Teaching assistant (24h per academic year)</em>
                </p>
            </div>
        </div>

    </div>


    <!-- Colonne de gauche / Publications ----> 
    <div class="colonne">
        <div class="block">
            <h1>Publications</h1>
            
            <p class="description">
                [S.1] Szatkownik, A., Decelle, A., Seoane, B., Béreux, N., Charpiat, G., Yelmen, B., Jay, F., Furtlehner, C. (2025).
                <em><a target="_blank" href="https://arxiv.org/abs/2510.24233">PRIVET: PRIVacy metric based on Extreme value Theory.</a> arXiv.</em> 2510.24233
            </p><br>
        
        
            <p class="description">
                [P.2] Szatkownik, A., Planche, L., Demeulle, M., Chambe, T., Ávila-Arcos, M.C., Huerta-Sanchez, E., Furtlehner, C., Charpiat, G., Jay, F., Yelmen, B. (2024). <em><a target="_blank" href="https://doi.org/10.1101/2024.10.28.620648">Diffusion-based artificial genomes and their usefulness for local ancestry inference.</a> bioRxiv.</em> DOI: 10.1101/2024.10.28.620648
            </p><br>
        
        
            <p class="description">
                [P.1] Szatkownik, A., Furtlehner, C., Charpiat, G., Yelmen, B., Jay, F. (2024). <em><a target="_blank" href="https://doi.org/10.1101/2024.08.07.607012">Latent generative modeling of long genetic sequences with GANs. </a> bioRxiv.</em> DOI: 10.1101/2024.08.07.607012
            </p><br>
        
            <p class="description">
                [C.1] Szatkownik, A., Furtlehner, C., Charpiat, G., Yelmen, B., Jay, F. (2024). <em><a target="_blank" href="https://proceedings.mlr.press/v240/szatkownik24a.html">Towards creating longer genetic sequences with GANs: Generation in principal component space.</a> Proceedings of the 18th Machine Learning in Computational Biology meeting, in Proceedings of Machine Learning Research,</em> 240:110–122
            </p><br>
        
            <p class="description">
                [J.2] Yelmen, B., Decelle, A., Boulos, L.L., Szatkownik, A., Furtlehner, C., Charpiat, G., Jay, F. (2023). <em><a target="_blank" href="https://doi.org/10.1371/journal.pcbi.1011584">Deep convolutional and conditional neural networks for large-scale genomic data generation.</a> PLOS Computational Biology,</em> Vol. 19, Issue 10: e1011584. DOI: 10.1371/journal.pcbi.1011584
            </p><br>
        
        
            <p class="description">
                [J.1] Szatkownik, A., Javier Zea, D., Richard, H., Laine, E. (2023). <em><a target="_blank" href="https://doi.org/10.1016/j.jsb.2023.107997">Building alternative splicing and evolution-aware sequence-structure maps for protein repeats.</a> Journal of Structural Biology,</em> DOI: 10.1016/j.jsb.2023.107997
            </p><br>
            
            <p class="small">C=CONFERENCE, J=JOURNAL, S=IN SUBMISSION, P=PREPRINT</p>
        </div>
    </div>

    <!-- Colonne de gauche / Research activities - Skills ----> 
    <div class="colonne">
        <div class="block">
            <h1>Research activities</h1>
            <div class="line">
                <ul class="description list">
                    <p>TALKS</p>
                    <li>LEGEND: Machine Learning for Evolutionary Genomics Data, Aussois, France, 2025.</li>
                    <li>Journée anniversaire des 10 ans de Sciences Animales Paris-Saclay, INRAE, Jouy-en-Josas, France, 2025.</li>
                    <li>Séminaire TAU, LISN, Gif-sur-Yvette, France, 2025.</li>
                    <li>Symposium on Synthetic clinical data, Queen Mary’s DERI, London, United Kingdom, 2025.</li>
                    <li>Séminaire MaIAGE, INRAE, Jouy-en-Josas, France, 2025.</li>
                    <li>Human Frontier Science Program meeting (HFSP), online, 2024.</li>
                    <li>LEGEND: Machine Learning for Evolutionary Genomics Data, Heraklion, Greece, 2024.</li>
                    <li>Machine Learning for Genomics <em><a href="https://gt-lego.cnrs.fr/">(LEGO)</a></em>, Lille, France, 2023.</li>
                    <li>Séminaire BioInfo, LISN, Gif-sur-Yvette, France, 2023.</li>
                    <li>Journée des Doctorants, LISN, Gif-sur-Yvette, France, 2023.</li>
                    <li>yr2biomodel : second edition of the Young Researchers’ Meeting at Sorbonne Université, Paris, France, 2022.</li>
                </ul>
            </div>
            <div class="line">
                <ul class="description list">
                    <p>POSTERS</p>
                    <li>Northern Lights Deep Learning (NLDL), Tromsø, Norway, 2024.</li>
                    <li>Machine Learning in Computational Biology (MLCB), Seattle, United States, 2023.</li>
                    <li>Junior Conference on Data Science and Engineering (JDSE), Orsay, France, 2023.</li>
                    <li>Alphy & AIEM, Grenoble, France, 2023.</li>
                    <li>JOBIM, Rennes, France, 2022.</li>
                </ul>
            </div>
            <div class="line">
                <ul class="description list">
                    <p>REVIEWING</p>
                    <li>MLCB (also part of scientific Program Committee for 2025)</li>
                </ul>
            </div>
            <div class="line">
                <ul class="description list">
                    <p>SUPERVISION</p>
                    <li>Ilan Aliouchouche, Master student, Université Paris-Saclay</li>
                    <li>Titouan Chambe and Maïwen Demeulle, Master student, Université Paris-Saclay</li>
                </ul>
            </div>
            <div class="line">
                <ul class="description list">
                    <p>SEMINARS</p>
                    <li>Co-organizer of LISN BioInfo’s team seminar</li>
                </ul>
            </div>
        </div>
    </div>


    <div class="colonne print">
        <div class="block">
            <h1>Softwares</h1>

            <div class="line">
                <ul class="description list">
                    <li> <a href="https://github.com/antoineszatkownik/PRIVET">PRIVET</a>: algorithm to detect instances of memorization and overfitting across diverse data modalities, including settings with very high dimensionality and limited sample sizes.</li>
                    
                    <li> <a href="https://gitlab.inria.fr/ml_genetics/public/artificial_genomes">Latent generative models and evaluation metrics for artificial genomes (SNP sequences)</a>: PCA-WGAN, PCA-DDPM, k-SNPs motifs.</li>
                    
                    <li> <a href="https://github.com/PhyloSofS-Team/aspring">ASPRING</a>: algorithm for evolutionary splicing graph objects to detect similar exonic regions that are modulated in number and composition by alternative splicing (Co-developed by Diego Javier Zea).</li>
                </ul>
            </div>
        </div>
    </div>


    <div class="colonne print">
        <div class="block">
            <h1>Skills</h1>
            
            <div class="line">
                <ul class="description list">
                    <li> <strong>Programming:</strong> Python, R, Bash, Slurm, Git</li>
                    
                    <li> <strong>Languages:</strong> French (Native), English (Fluent)</li>
                    
                    <li> <strong>Interests:</strong> Sculpture (clay, modeling paste, steel, welding), Painting (acrylic), Drawing (charcoal), Collage, Computer graphics, Cinema, Writing (lino, fanzine, poetry)</li>
                </ul>
            </div>
        </div>
    </div>


</main>


 <footer>
    <div><input type="checkbox" id="mentions-legales"  hidden />
          <label for="mentions-legales">
            <p class="span-nochecked">OPEN LEGAL NOTICE</p>
            <p class="span-checked">CLOSE LEGAL NOTICE</p>
          </label>
            <div class="legal">
                <div>
                    Legal notices<br><br>
                    Publisher's information<br>
                    — Antoine Szatkownik<br>
                    75014, Paris, France<br>
                    <a href="mailto:szatkownik@lri.fr">szatkownik@lri.fr</a>
                    <a href="https://www.antoineszatkownik.com/">https://www.antoineszatkownik.com/</a>
                </div>

                <div>
                    Hosting information<br>
                    — OVH<br>
                    2 rue Kellermann<br>
                    59100 Roubaix - France<br>
                    SAS au capital de 10 174 560 €<br>
                    RCS Lille Métropole 424 761 419 00045<br>
                    Code APE 2620Z<br>
                    <a href="https://www.ovhcloud.com/fr/">https://www.ovhcloud.com/fr/</a>
                </div>

                <div>
                    Graphic desinger & developer information<br>
                    — Milène Laforge<br>
                    75014, Paris, France<br>
                    <a href="mailto:hello@milenelaforge.com">hello@milenelaforge.com</a>
                    <a href="https://www.milenelaforge.com/">https://www.milenelaforge.com/</a>
                </div>
            </div>
    </div>
    <div><p>2025</p></div>
    <div><p>DESIGN & CODE: <a href="https://www.milenelaforge.com/" target="blank">Milène Laforge</a></p></div>

</footer>


</body>
</html>